src/dbus.c

```diff
@@ -85,6 +85,10 @@ static const char *introspection_xml =
     "            <annotation name=\"org.freedesktop.DBus.Property.EmitsChangedSignal\" value=\"true\"/>"
     "        </property>"
 
+    "        <property name=\"displayed\" type=\"u\" access=\"read\" />"
+    "        <property name=\"history\" type=\"u\" access=\"read\" />"
+    "        <property name=\"waiting\" type=\"u\" access=\"read\" />"
```

Comment:
Similar here, I'd name it `displayedLength` to avoid naming conflicts for future expansions.

Reply:
I would name them all in the same way:
* `displayedLength`
* `historyLength`
* `waitingLength`

Without the technical knowledge, it's hard to get, what those properties shall return. Maybe we can find names?

Reply:
I gave an example for displayed but I did mean to change all of them :). Your list of names looks good to me.